p_soc_auto/settings.py

```diff
@@ -159,7 +159,7 @@
 CELERY_BROKER_URL = 'amqp://guest:guest@localhost//'
 
 CELERY_ACCEPT_CONTENT = ['json', 'pickle']
-CELERY_RESULT_BACKEND = 'django-db'
+CELERY_RESULT_BACKEND = 'rpc://'
```

Comment:
CELERY_RESULT_BACKEND is used to store back-end results
we could have used django-db/file path...etc
how do we view rpc://

Reply:
the celery documentation strongly advises against using django-db as the backend. in our own implementation, we were running out of disk space because of using the database to store the results. with rpc:// we can view the results using celery flower. my intention is to fully disable storing the results and just using record time stamps to make sure that tasks are executed on time